backend/main.go

```diff
@@ -235,7 +235,18 @@ func main() {
 			highlight.SetOTLPEndpoint("http://collector:4318")
 		}
 	}
-	highlight.Start()
+
+	serviceName := string(runtimeParsed)
+	if runtimeParsed == util.Worker {
+		if handlerFlag != nil {
+			serviceName = *handlerFlag
+		}
+	}
+
+	highlight.Start(
+		highlight.WithServiceName(serviceName),
```

Comment:
This is `all` locally
![Screenshot 2023-07-28 at 1 11 39 PM](https://github.com/highlight/highlight/assets/58678/55366a31-14d5-4f37-bedd-06f6a877a2fd)

but will change to use the `runtimeFlag` in prod:

![Screenshot 2023-07-28 at 1 02 47 PM](https://github.com/highlight/highlight/assets/58678/56724cc3-a2ce-44f0-bb43-4ba1da0cdcab)